test/rules/noUnusedVariableRuleTests.ts

```diff
@@ -113,4 +113,66 @@ describe("<no-unused-variable>", () => {
 
         assert.lengthOf(actualFailures, 0);
     });
+
+    describe("with react option", () => {
+        function pass(fileName: string, reactOptSet: boolean) {
```

Comment:
also, please move all helper methods to the bottom of this describe block. Test suites should be read first, then helper methods later.